installer.sh

```diff
@@ -92,16 +108,18 @@ generate() {
     if ! is_target_file_set; then 
         return 1 
     fi
-    if [ -z $BROWSER ]; then 
+    if [ -z "$BROWSER" ]; then 
         select_browser
     fi
 
-
     rm -rf "$BUILD_DIR"
+    set_sys_opts
```

Comment:
This now does what it says on the tin, and does it from the logical place to do it.